### DISCUSSION TITLE
[blocks]: Chat Interface frontend with support for FastAPI or Vercel AI SDK

### DISCUSSION THREAD

**Discussion by chirag-mehta** [shadcn-ui/ui#3172] [Blocks Request]
### Description

Chat Interface which supports streaming outputs and even transparency into agent actions. A conversations sidebar would be greatly helpful.

### Example

chat.openai.com

**Comment by jakobhoeg** [comment]
@chirag-mehta If you need inspiration, check out my repo [here](https://github.com/jakobhoeg/nextjs-ollama-llm-ui).
